Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.11 as build
```

Comment:
I normally use the image tag that is as precise as possible ie in this case`1.11.2-alpine3.8`. I could live with `1.11-alpine3.8` so fixes are included when rerunning docker builds.